src/moore/static/sass/partials/pages/_involvement.scss

```diff
@@ -1,6 +1,6 @@
 .positions {
   .collapsible-header {
-    height: 60px;
+    height: 80px;
```

Comment:
This is not the right solution. It seems the headers have an added white space on top, that should be removed.

Reply:
I couldn't find that fix. Put in a request a while ago. If it's an easy fix I would prefer som help from you or Johan.

Reply:
@fellab I've seen a few issues that I could help out with, but I'm still settling down here in Melbourne. I'll be helping out then. In the meantime @jsnider should be able to solve at least this CSS issue.

Reply:
Can you help out with this @jsnider ?